src/main/java/ProjectOcean/Model/Course.java

```diff
@@ -173,10 +173,11 @@ public boolean equals(Object o) {
     /**
      * @return a hash code
      */
+    /*
     @Override
     public int hashCode() {
         return Objects.hash(courseCode, courseName, studyPoints, studyPeriod, examiner, examinationMeans, language, requiredCourses, coursePMLink, courseDescription);
-    }
+    } */
```

Comment:
why leave this out?

Reply:
Because if we use it we cannot place the same course in two places since the HashMap sees them as the same object. I have now removed this to avoid confusion.